packages/features/webhooks/components/WebhookTestDisclosure.tsx

```diff
@@ -15,15 +15,11 @@ export default function WebhookTestDisclosure() {
   });
 
   return (
-    <div className="space-y-0 rounded-md border-0 border-neutral-200 bg-white sm:mx-0 md:border">
-      <div className="flex justify-between border-b p-4">
-        <div className="flex items-center space-x-1">
-          <h3 className="font-sm self-center font-medium text-black">{t("webhook_response")}</h3>
-          {mutation.data && (
-            <Badge variant={mutation.data.ok ? "green" : "red"}>
-              {mutation.data.ok ? t("passed") : t("failed")}
-            </Badge>
-          )}
+    <>
+      <div className="flex justify-between">
+        <div>
+          <p className="font-medium text-black">{t("webhook_test")}</p>
+          <p className="font-sm mb-4 text-gray-600">{t("test_webhook")}</p>
```

Comment:
```suggestion
          <p className="mb-4 text-sm font-normal text-gray-500">{t("test_webhook")}</p>
```

Reply:
Hey, @CarinaWolli in figma its 600.

But visually 500 looking similar. So what should I go with 600 or 500?

![image](https://user-images.githubusercontent.com/69904519/215531902-1ad1f096-5372-40ae-98a3-90a3642582ba.png)

Reply:
@joeshajan Sorry that's right 👍 still `font-sm` should be `text-sm` here

Reply:
Yeah sure